src/schema/bookmarks.ts

```diff
@@ -385,20 +438,11 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         removeNonPublicThresholdSquads: false,
       },
     ),
-    bookmarkLists: (
-      source,
-      args,
-      ctx: AuthContext,
-      info,
-    ): Promise<GQLBookmarkList[]> =>
-      ctx.loader.loadMany<BookmarkList>(
-        BookmarkList,
-        { userId: ctx.userId },
-        info,
-        {
-          order: { name: 'ASC' },
-        },
-      ),
+    bookmarkLists: (_, __, ctx: AuthContext): Promise<GQLBookmarkList[]> =>
+      ctx.con.getRepository(BookmarkList).find({
```

Comment:
I think we should use the graphorm function here.

Reply:
Yes, on reads it should always be the case.
Probably actually also can just read from RR here. 

CC @capJavert we talked about this during RFC stages, but missed it as well.

Reply:
Updated ✔️ 

@rebelchris where I can find a sample of using replica? 👀

Reply:
@ilasw look for `queryReadReplica` in code

Reply:
https://github.com/dailydotdev/daily-api/blob/cc174bcecdad2cac05371d2e1be854337bc5aea5/src/schema/comments.ts#L736

![Screenshot 2024-12-05 at 11 39 41](https://github.com/user-attachments/assets/7bb8d49c-bb1f-4bf1-bc1f-c87f101adf80)

Last prop is readReplica :)

Reply:
Updated, thanks! ✔️